src/robusta/core/model/services.py

```diff
@@ -21,4 +24,7 @@ def __eq__(self, other):
             and self.namespace == other.namespace
             and self.classification == other.classification
             and self.deleted == other.deleted
+            and sorted(self.images) == sorted(other.images)
```

Comment:
what about old data that don't contain the images?

Reply:
same for the labels

Reply:
It works good.
Because on the old data it will have empty array, on the new it will have the image list.
The equals method is used to check if the service changed, and if so it should be persisted to the DB.

So here, an old service will be persisted again with the images